src/util/DataUtils.ts

```diff
@@ -72,7 +72,7 @@ export const getAnyElementOfObject = (obj: any) => {
   return null;
 };
 
-export const hasDuplicate = (ary: Array<unknown>) => {
+export const hasDuplicate = (ary: Array<any>) => {
```

Comment:
why do we need to change this? Probably better to keep it `unknown` no?

Reply:
oh nevermind I lost context since I looked at this. Can't be used as an index type, got it